.editorconfig

```diff
@@ -1,15 +1,34 @@
+# https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties
 root = true
 
 [*]
 charset = utf-8
-end_of_line = lf
-insert_final_newline = true
 indent_style = space
 indent_size = 4
+end_of_line = lf
 trim_trailing_whitespace = true
+insert_final_newline = true
+
+# Unofficial config, support depends on the application reading the file.
+# https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#supported-by-a-limited-number-of-editors
+# https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#ideas-for-domain-specific-properties
+max_line_length = off
+quote_type = single
+spaces_around_operators = true
+spaces_around_brackets = none
+
+# https://en.wikipedia.org/wiki/Indent_style
+indent_brace_style = Allman
```

Comment:
This isn't conform PSR? This should be K&R I think?

Reply:
You're absolutely right